src/internal/connector/onedrive/collections.go

```diff
@@ -325,6 +328,33 @@ func (c *Collections) Get(
 		folderPaths[driveID] = map[string]string{}
 		maps.Copy(folderPaths[driveID], paths)
 
+		// Set all folders in previous backup but not in the current
+		// one with DoNotMerge
+		if c.ctrl.ToggleFeatures.EnableOneDriveDeltaIncrementals && delta.Reset {
+			for i, p := range oldPaths {
```

Comment:
`for i, p` is confusing when `oldPaths` is a map; `i` suggests an index, as if oldPaths is an array.
```suggestion
			for k, p := range oldPaths {
			// alt: for folderID, p := range oldPaths {
```